java/src/main/java/com/google/appengine/tools/mapreduce/MapReduceJob.java

```diff
@@ -138,28 +119,13 @@ public static <I, K, V, O, R> String start(
     throw new UnsupportedOperationException("Use PipelineOrchestrator:start");
   }
 
-  private static void verifyBucketIsWritable(MapReduceSettings settings) {
```

Comment:
not used

Reply:
yeah ... left thinking we could recover that check at some point, but prob ok to drop